### DISCUSSION TITLE
Autopilot idea: extra functions

### DISCUSSION THREAD

**Discussion by synthgloom** [ppy/osu#23001] [Ideas]
Reasoning for this suggestion: Autopilot can be quite helpful when mapping (to check how certain sections will/could be performed by players), and getting a frame of reference whether (and how) a really difficult section in a map is possible to play. However, because Autopilot is too perfect, it doesn't support these use cases as well as it could.

My suggestion: add extra functions to Autopilot that can be toggled on/off in the mod's context menu.

Features:
**Force Alternate Tapping** - forces Autopilot to alternate K1 and K2, instead of defaulting to singletap
**Realistic Aim** - currently, Autopilot moves the cursor between two notes perfectly: it accounts for the time and distance between notes, and does a perfectly smooth movement into the center of the hit object at a consistent speed for all the notes in the map. However, humans can't achieve such precise "flow aim". Changing the speed at which the cursor travels between notes, and changing the accuracy (how close to the center of the hit object the cursor lands) might make Autopilot's aim a more realistic emulation of real gameplay.

**Comment by peppy** [comment]
i'm pretty sure the first suggestion can already be applied via another mod.
